### ISSUE TITLE
Path "/filebrowser_filer/filer_version/" was changed and jQuery.get is returning 404

### ISSUE THREAD

**Issue by peca11** [nephila/django-ckeditor-filebrowser-filer#59]
I think this path "/filebrowser_filer/filer_version/" changed to "/filebrowser_filer/version/" 

The current call returns a 404

File: 
https://github.com/nephila/django-ckeditor-filebrowser-filer/blob/17a3f35681e0f723e499c35c43249629d1ef025a/ckeditor_filebrowser_filer/static/ckeditor/ckeditor/plugins/filerimage/plugin.js#L43

It should change to 
`jQuery.get('/filebrowser_filer/version/', { }, function(data) {
`

**Comment by fivkovic-dev** [comment]
I have encountered the same issue.

**Comment by ershadul** [comment]
A temporary workaround is:
```    
url(r'^filebrowser_filer/filer_version/$', RedirectView.as_view(url='/filebrowser_filer/version/', permanent=True)),
```

**Comment by protoroto** [comment]
@peca11 @ershadul @fivkovic-dev Thanks for poiting this out! I can't find when did that url change (it looks like it was always under `/version/`): do you happen to know when did this happen? 
Anyway, I've opened a pr, I hope to release a new version as soon as possibile.

**Comment by protoroto** [comment]
@peca11 @ershadul @fivkovic-dev We've released version 0.5.0 , check if this error is no longer there now.

**Comment by peca11** [comment]
@protoroto Many thanks! It's working :)
I don't know for sure when it started happening.
